### PR TITLE
LPS-147075 Do not create a mask when previewing a Fragment Collection

### PR DESCRIPTION
**Pull request by p2kmgcl**

### CODE REVIEW

File: modules/apps/style-book/style-book-web/src/main/resources/META-INF/resources/js/fragment-collection-preview/FragmentCollectionPreview.js [hunk 1]
```diff
@@ -13,17 +13,41 @@
...
-export default function FragmentCollectionPreview({fragments, namespace}) {
+const FRAGMENT_COLLECTION_BLOCKLIST = {
+	BASIC_COMPONENT: [
+		'BASIC_COMPONENT-external-video',
+		'BASIC_COMPONENT-html',
+		'BASIC_COMPONENT-separator',
+		'BASIC_COMPONENT-spacer',
+		'BASIC_COMPONENT-video',
+	],
+};
```
Comment (victorg1991) [inline modules/apps/style-book/style-book-web/src/main/resources/META-INF/resources/js/fragment-collection-preview/FragmentCollectionPreview.js:30]: 🕯️ to those brave fragments that didn't make it

Comment (p2kmgcl) [reply modules/apps/style-book/style-book-web/src/main/resources/META-INF/resources/js/fragment-collection-preview/FragmentCollectionPreview.js:30]: they don't deserve it, they are ugly

File: modules/apps/style-book/style-book-web/src/main/resources/META-INF/resources/js/style-book-editor/StyleBookEditor.js [hunk 2]
```diff
@@ -40,6 +40,11 @@ const StyleBookEditor = ({
 			? getMostRecentLayout(config.previewOptions)
 			: initialPreviewLayout
 	);
+	const [previewLayoutType, setPreviewLayoutType] = useState(
+		config.previewOptions.find((type) =>
```
[the remark this answers is not in the document]
Comment (p2kmgcl) [reply modules/apps/style-book/style-book-web/src/main/resources/META-INF/resources/js/style-book-editor/StyleBookEditor.js:44]: Porras you are right

**Comment by p2kmgcl** [comment]
ci:test:relevant

**Comment by victorg1991** [comment]
Re-chévere!

**Comment by p2kmgcl** [comment]
:unicorn:

**Comment by ealonso** [comment]
ci:forward

**Pull request closed without merging**